templates/core/about/builds.html

```diff
@@ -3,77 +3,106 @@
-		<code>Cargo.toml</code>. If a crate doesn't have this field,
-		no README will be displayed.
-	</p>
-
-	<h3>Diagnosing a failed build</h3>
-
-	<h4>Missing dependencies</h4>
-	<p>
-		Missing dependencies are a common reason for a failed build.
-		Docs.rs dependencies are managed through
-		<a href="https://github.com/rust-lang/crates-build-env">crates-build-env</a>;
-		see <a href="https://forge.rust-lang.org/docs-rs/add-dependencies.html">Forge</a> for instructions
-		on how to make a PR. You can always <a href="{{ docsrs_repo | safe }}/issues">file an issue</a>
-		if you're having trouble.
-	</p>
-
-	<h4>Detecting Docs.rs from <code>build.rs</code></h4>
-	<p>
-		Docs.rs builds crates with the environment variable <code>DOCS_RS</code> set to
-		<code>1</code>, which enables the crate to detect docs.rs and build the
-		crate differently. This can be helpful if you need
-		dependencies for building the library, but not for building the documentation.
-	</p>
-
-	<h4>Detecting Docs.rs from <code>#[cfg]</code> attributes</h4>
-	<p>
-		You can detect Docs.rs by having a <a href="https://doc.rust-lang.org/cargo/reference/features.html">feature</a>
-		which is only set by Docs.rs. See <a href="metadata">Metadata</a> for more information.
-	</p>
-
-	<h4>Global sandbox limits</h4>
-
-	<p>
-		All the builds on docs.rs are executed inside a sandbox with limited
-		resources. The current limits are the following:
-	</p>
-
-	{{ macros::crate_limits(limits=limits) }}
-
-	<p>
-		If a build fails because it hit one of those limits please
-		<a href="{{ docsrs_repo | safe }}/issues/new/choose">open an issue</a>
-		to get them increased for your crate.
-		Note that network access will not be enabled for any crate.
-	</p>
-
-	<h4>Test crate documentation build locally</h4>
-	{%- set build_subcommand = docsrs_repo ~ "/blob/master/README.md#build-subcommand" -%}
-	<p>
-		The <a href="{{ build_subcommand | safe }}">docs.rs README</a> describes how to build an
-		unpublished crate's documentation locally using the same build environment as the build agent.
-	</p>
-	</div>
+    {%- set docsrs_repo = "https://github.com/rust-lang/docs.rs" -%}
+    <h1>Builds</h1>
+    <div class="container about">
+    <p>
+        Docs.rs automatically builds documentation for crates released on <a href="https://crates.io/">crates.io</a>.
+         
+        It may take a while to build your crate, depending on how many crates are in <a href="/releases/queue">the queue</a>.
+    </p>
+
+    <p>
+        All crates are built in a sandbox using the nightly release of the Rust compiler.
+        {%- if rustc_version %}
+        The current version in use is <code>{{ rustc_version }}</code>.
+        {%- endif -%}
+    </p>
+
+    <h3>Notes on using Docs.rs</h3>
+
+    <h4>Setting a README</h4>
+    <p>
+        The README of a crate is taken from the <code>readme</code> field defined in
+        <code>Cargo.toml</code>. Omitting this field means that no README will be displayed.
```

Comment:
nit: this uses a little more complicated language which is more concise but might be harder for some readers. Is there a reason you changed it?